### PR TITLE
Add crowdin file

### PR DESCRIPTION
**Pull request by daniserejo**
Add crowdin file

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
+- `crowdin.file` for crowdin integration.
```
Comment (camilavcoutinho) [inline CHANGELOG.md]:
```suggestion
- Crowdin.file for crowdin integration.
```

**Review by camilavcoutinho — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
+- Crowdin.file for crowdin integration.
```
Comment (camilavcoutinho) [inline CHANGELOG.md]:
```suggestion
- Crowdin configuration file.
```

**Review by camilavcoutinho — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
```
Comment (salesfelipe) [inline CHANGELOG.md]: Should this be made to the version 1.x aswell?

File: CHANGELOG.md (hunk 2 above)
Comment (salesfelipe) [reply CHANGELOG.md]: I don't think you need changelog for this

File: CHANGELOG.md (hunk 3 above)
Comment (daniserejo) [reply CHANGELOG.md]: @camilavcoutinho

**Review by camilavcoutinho — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+
+- Crowdin configuration file.
+
```
Comment (salesfelipe) [inline CHANGELOG.md]:
```suggestion
```

**Review by salesfelipe — APPROVED** [review]

**Pull request merged**